.gitlab-ci.yml

```diff
@@ -166,7 +166,10 @@ variables:
   DATADOG_AGENT_NIKOS_BUILDIMAGES: v20878799-a2f77ae
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v20878799-a2f77ae
+  # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
+  # and check the job creating the image to make sure you have the right SHA prefix
   TEST_INFRA_DEFINITIONS_BUILDIMAGES: 2c1b7c55a327
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should we have a check to avoid merging a non empty suffix on main ?

Reply:
I think it's a good idea, do you want that as part of the `do-no-merge` job, or should I create a specific one ?

Reply:
Done in `do-not-merge` job